examples/paper_experiments/README.md

```diff
@@ -0,0 +1,53 @@
+This folder contains the data processing and code to reproduce the graphstorm paper results.
+
+# Data Processing 
+## Amazon Review
+### Process raw dataset into parquet files
+We first download the full Amazon Review dataset from https://cseweb.ucsd.edu/~jmcauley/datasets/amazon_v2/. 
+Review data is in https://datarepo.eng.ucsd.edu/mcauley_group/data/amazon_v2/categoryFiles/All_Amazon_Review.json.gz
+Meta data is in https://datarepo.eng.ucsd.edu/mcauley_group/data/amazon_v2/metaFiles/All_Amazon_Meta.json.gz
+```
+wget https://datarepo.eng.ucsd.edu/mcauley_group/data/amazon_v2/metaFiles2/meta_Video_Games.json.gz \
+-P raw_data/
+```
+
+After you unzip the above two files, we generate the parquet files as input of graph construction step. 
+```
+python construct_AR_items.py
+python construct_AR_reviews.py
+```
+
+### Run gconstruct on process_data
+In the first step, the processed data are under processed_data/amazon_review/
+Once the data are processed, run the following command to construct a graph
+for node classification and link prediction tasks. 
+Recommended machine: x1e.32xlarge	
```

Comment:
We may need to add a link for x1e.32xlarge here for reference if anyone wants to refer the instance type for x1e.32xlarge. https://aws.amazon.com/ec2/instance-types/x1e/

Reply:
Can you profile the maximum memory requirement to process the data?